### DISCUSSION TITLE
Difference between owner and creater

### DISCUSSION THREAD

**Discussion by WeT-Klb** [nocodb/nocodb#4022] [Q&A] [answered]
Please enter the following details
Node: v12.22.6
Arch: x64
Platform: linux (QNAP 5.0.0.2131)
Docker: true
Database: mysql2 (MariaDB10.5)
ProjectOnRootDB: true
RootDB: mysql2 (MariaDB 10.5)
PackageVersion: 0.96.4```

**Steps To Reproduce**

1. In one project there is one "owner" and one "creator"
2. Seems i can change "owner" to whatever i want - but can´t change any role to "owner"
3. See error

**Expected Behavior**

Why "owner" can´t be defined anymore?

**Screenshots**

![Nocodb1](https://user-images.githubusercontent.com/20600342/189841264-0dce6641-e969-42ea-9d86-ad51acdddb66.jpg)
![Nocodb2](https://user-images.githubusercontent.com/20600342/189841292-a4fd7e14-9333-45cc-8208-e4bd4e425efc.jpg)

Join our discord : https://discord.gg/5RgZmkW for realtime help.

**Comment by o1lab** [comment]
@pranavxc

**Comment by pranavxc** [comment]
Both `owner` and `creator` have the same permission, the only difference is `owner` is the one who created the project.

**Comment by WeT-Klb** [comment]
Ok - but then how an old project can have two owners if only one created the project?
![Nocodb3](https://user-images.githubusercontent.com/20600342/189897420-0a91e493-700b-48f3-8bae-ddf786faab62.jpg)

Or did those roles change in the past?

**Comment by wingkwong** [comment, marked as answer]
@WeT-Klb I believe we've made some changes on roles. Here's the current behaviours.

- Currently only one owner in the project. You cannot invite people with owner role.

<img width="216" alt="image" src="https://user-images.githubusercontent.com/35857179/195260061-6f0a9ccd-b01e-41d0-b375-70300eb6b49d.png">

- You cannot change the role of the owner.

<img width="531" alt="image" src="https://user-images.githubusercontent.com/35857179/195259907-5ae23dd8-3946-4991-9d0e-8f327eac8296.png">
